src/main/webapp/app/routes.tsx

```diff
@@ -31,7 +31,7 @@ const Routes = () => (
       <ErrorBoundaryRoute exact path="/auth/email/reset" component={AuthEmailReset} />
       <PrivateRoute exact path="/profile" component={UserProfile} />
       <PrivateRoute path="/admin" component={Admin} hasAnyAuthorities={[AUTHORITIES.ADMIN]} />
-      <PrivateRoute path="/entity" component={Entities} hasAnyAuthorities={[AUTHORITIES.USER]} />
+      <ErrorBoundaryRoute path="/entity" component={Entities} />
```

Comment:
can change back to PrivateRoute?

Reply:
Resolved in fe845d284e2cb16f075baa6dbb9d9c1eba3efa68